### ISSUE TITLE
1.1.6 is missing minified dist css

### ISSUE THREAD

**Issue by ptondereau** [YUzhva/react-leaflet-markercluster#28]
Using ES6 (Babel) syntax with 1.1.6 leads to this error : 

```
Uncaught Error: Cannot find module 'react-leaflet-markercluster/dist/style.min.css' from 'react-leaflet-markercluster/dist/react-leaflet-markercluster.min.js'
```

**Comment by YUzhva** [comment]
Hey hi @ptondereau 

In v1.1.6 lib depencencies moved from `dependencies` to `peerDependencies`.

Try to rebuild node packages:
```bash
npm rebuild
```

**Comment by ptondereau** [comment]
Thank you for quick response.
unfortunately it didn't solved this issue

**Comment by ptondereau** [comment]
btw, modules folder look like this
<img width="467" alt="capture d ecran 2017-07-12 a 12 10 39" src="https://user-images.githubusercontent.com/4287777/28112981-2f909050-66fb-11e7-84fe-17776669f1b0.png">

**Comment by YUzhva** [comment]
dist folder looks full at master branch: https://github.com/YUzhva/react-leaflet-markercluster/tree/master/dist

Maybe try to reinstall plugin:
```
rm -rf node_modules/react-leaflet-markeruster

yarn add react-leaflet-markeruster
```

Give me 5 min, I'll try to create new project with create-react-app and would double-check it.

**Comment by ptondereau** [comment]
Okay, I found a way...
![capture d ecran 2017-07-12 a 12 17 45](https://user-images.githubusercontent.com/4287777/28113253-2ac714e4-66fc-11e7-886e-b4256be9bc81.png)

**Comment by ptondereau** [comment]
Is there any chance to backport your lib to ~15.4 ?

**Comment by YUzhva** [comment]
That is just a peer dependency, that should no stop your code from running.

I just test it and I'm able to use `react-leaflet-markeruster` with `react@15.4.2`.

**Comment by ptondereau** [comment]
I did:

```
$ rm -rf node_modules && yarn install --force
```

Nothing changed :/

**Comment by ptondereau** [comment]
as this problem has nothing to do with this lib, I close it. Thanks for help

**Comment by YUzhva** [comment]
@ptondereau 

What version of yarn are you using (maybe you need to update it)?

Did you try to install dependencies with npm:
```
$ rm -rf node_modules && npm install
```

**Comment by ptondereau** [comment]
Files are correctly generated but continue to show same error in console no matter if it installed with yarn or npm

**Comment by ptondereau** [comment]
I found the problem: CSS module

**Comment by YUzhva** [comment]
Thank you, for the answer.
This issue with importing styles will be resolved in release `v1.2.0`

Just need to update doc and add depreciation/warning console messages, so other could switch from old props to new...

**Comment by aqellez** [comment]
I have same error with importing, also tryid to rebuild and reinstall with both package managers, but still have this error.
